### PR TITLE
FPWD preparation

### PR DESCRIPTION
**Pull request by simoneonofri**
as https://github.com/w3c/transitions/issues/647

 - update index.bs with metadata
 - update SVGs
 - update w3c.json

### CODE REVIEW

**Comment by npm1** [comment]
What are the svg updates? I don't see a noticeable difference in the images

File: spec/index.bs [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-Shortname: FedCM
+Shortname: fedcm
 Level: 1
-Status: w3c/CG-DRAFT
-Group: fedidcg
-ED: https://fedidcg.github.io/FedCM/
-Repository: fedidcg/FedCM
-Editor: Nicolás Peña Moreno, Google Inc. https://google.com, npm@chromium.org
-Former Editor: Sam Goto, Google Inc. https://google.com, goto@google.com
+Status: w3c/FPWD
```
Comment (npm1) [inline spec/index.bs:5]: I didn't know FPWD is a thing? When would this change to WD?

File: spec/index.bs [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+Group: fedid
+TR: http://www.w3.org/TR/fedcm/
```
Comment (npm1) [inline spec/index.bs:7]: This does not exist currently

**Comment by simoneonofri** [comment]
some attributes added by the editor and the image ids.

this happened because  they are embedded in the HTML

File: spec/index.bs (hunk 1 above)
Comment (simoneonofri) [reply spec/index.bs:5]: FPWD is a First Public Working Draft :)

File: spec/index.bs (hunk 2 above)
Comment (simoneonofri) [reply spec/index.bs:7]: waiting for an ack to create it

**Review by npm1 — APPROVED** [review]

**Review by samuelgoto — APPROVED** [review]
LGTM++

**Pull request merged**